pkgs/development/python-modules/hass-web-proxy-lib/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+}:
+
+buildPythonPackage {
+  pname = "hass-web-proxy-lib";
+  version = "0.0.7";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "dermotduffy";
+    repo = "hass-web-proxy-lib";
+    rev = "f96dfdec6e24275dc83b462a3471d89509f3d42a";
+    sha256 = "sha256-RJ7XUkgutgnbwZnmV7jtt+Hit7ZM/08hNZWTTEARlNc=";
+  };
+
+  build-system = [ poetry-core ];
+
```

Comment:
Have you tried enabling the test suite? I think this library can only work on the same python version as home-assistant, because it requires `(toPythonModule home-assistant)` as a test dependency.

Reply:
I got close, but I need to create another package for `pytest_homeassistant_custom_component` it seems